www-misc/buku/buku-4.1.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6} )
+
+inherit bash-completion-r1 distutils-r1
+
+DESCRIPTION="Powerful command-line bookmark manager"
+HOMEPAGE="https://github.com/jarun/Buku"
+SRC_URI="test? ( https://github.com/jarun/Buku/files/1319933/bookmarks.zip -> ${P}-bookmarks.zip )"
```

Comment:
Don't bind it to specific package version when the same file is shared by multiple versions. This creates absurd `buku-9999-bookmarks.zip`.